_events/2024-03-23-a-sample-event.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: event
+title: "OLS: a training and mentoring program for the open-research-curious"
+description: "Detailed event description"
+date: 2024-03-23
+time: "10:00 AM WAT"
+duration: 2 hours
+location: "Online"
+speakers:
+  - name: "Y.Y"
+  - affiliation: "Open Life Science"
```

Comment:
I am not sure about that. Maybe better to have something describing OLS participation

Reply:
Not sure I understand what you mean.

Reply:
What I meant is instead of having a `speakers` metadata, having something called `ols-involvement` with a list of team members that are involved and their type of involvements (e.g. organizer, speaker, etc). Something like

```
ols-involvement: 
- 
   member: 
   involvement: 
-
   member: 
   involvement: 
```

Is it more clear?